my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] `MyList` class implements #each method that yields all members of @list in succession

I know that ruby already implement that method but for a question of practice we recommend applying this method

```ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```